docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -60,6 +60,9 @@ This allows us to rely on [Hooks](https://reactjs.org/docs/hooks-intro.html).
 
 ### Styles
 
+- ⚠️ Material-UI depends on JSS v10. JSS v10 is not backward compatible with v9.
```

Comment:
I hope I can warn or throw about the issue. I have a codesandox to further improve the DX: https://codesandbox.io/s/py5pqo3y5q. It's the topic for another pull request.